### ISSUE TITLE
Pull Requests not showing with new Branches after fetching

### ISSUE THREAD

**Issue by tierninho** [desktop/desktop#3429]
After creating a new branch and PR on dotcom, and then fetching the changes in Desktop, only the new branch showed up. Desktop continued to say `0` pull requests in the list despite one PR existing in the branch.

<img width="740" alt="screen shot 2017-11-27 at 11 05 53 am" src="https://user-images.githubusercontent.com/14828183/33261428-076a7c74-d363-11e7-97c6-55de2eb69423.png">
<img width="795" alt="screen shot 2017-11-27 at 11 07 20 am" src="https://user-images.githubusercontent.com/14828183/33261498-38ce3008-d363-11e7-9b38-b6458c3c8aec.png">

Nothing suspicious in the logs. If you quit Desktop, switch to a different Repo and then switch back, or re-select the current Repo, then the PR will increment.

v1.0.10-beta2, mac

**Comment by iAmWillShepherd** [comment]
Pull requests are loaded on a set timer. Could it be the case that enough time was not given for this to happen? We also refresh the PR list when a user switches repositories, so that may explain why you saw it pop up after reselecting your repository...

@joshaber do you think it's a good idea to fetch PRs when the user manually fetches changes?

**Comment by joshaber** [comment]
>do you think it's a good idea to fetch PRs when the user manually fetches changes?

Definitely 👍 💯

**Comment by tierninho** [comment]
Pretty sure removing the timer will fix this... 

<img width="963" alt="screen shot 2017-11-28 at 1 46 16 pm" src="https://user-images.githubusercontent.com/14828183/33320457-9c5f5bee-d442-11e7-93fa-61f3caedc5d5.png">

I deleted a branch (on the remote too), and then clicked on the related Pull Request to surface the above error.

**Comment by iAmWillShepherd** [comment]
>I deleted a branch (on the remote too), and then clicked on the related Pull Request to surface the above error.

@tierninho, maybe that should be a separate issue?

**Comment by tierninho** [comment]
@iAmWillShepherd no problem, will retest after https://github.com/desktop/desktop/pull/3448 goes out 👍

**Comment by iAmWillShepherd** [comment]
Pulling in @joshaber's [comment](https://github.com/desktop/desktop/pull/3448#issuecomment-348231047)

**Comment by iAmWillShepherd** [comment]
@tierninho I've gone through your reproduction steps with the new work that I've done in PR #3503  and am no longer getting the same behavior, so this issue should be resolved when that lands.

Would you mind trying to get the issue to happen again?

**Comment by tierninho** [comment]
@iAmWillShepherd sure thing, once the new build lands, i'll be sure to test it ⚡️

**Comment by tierninho** [comment]
@iAmWillShepherd works like charm now, nice work!

**Comment by kasenvr** [comment]
Funnily enough, I am still getting this bug.

**Comment by tierninho** [comment]
@kasenvr Please fill out a new issue https://github.com/desktop/desktop/issues/new?template=bug_report.md if you'd like us to investigate. Thank you.

**Comment by faizaldong** [comment]
here 2021, I am still facing this issue.

before select branch the button PR is there but it disable
<img width="1397" alt="Screenshot 2021-07-12 at 1 34 59 AM" src="https://user-images.githubusercontent.com/17352624/125204767-73833700-e2b1-11eb-9507-956607c10ae9.png">

after select develop branch, the PR button its gone
<img width="1397" alt="Screenshot 2021-07-12 at 1 35 14 AM" src="https://user-images.githubusercontent.com/17352624/125204800-96155000-e2b1-11eb-9385-ebd894fb3d52.png">


@tierninho
